src/App.jsx

```diff
@@ -14,57 +15,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selected, setSelected] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selected ? (
+        <h1 className="title is-flex is-align-items-center">
+          {selected} is selected
+          <button
+            onClick={() => {
+              setSelected('');
```

Comment:
Consider moving this code to a separate function and reusing it further in the code